ui/src/plugins/dev.perfetto.RecordTraceV2/config/config_manager.ts

```diff
@@ -40,6 +40,8 @@ export class ConfigManager {
   private enabledProbes = new Map<string, boolean>();
   private indirectlyEnabledProbes = new Map<string, Set<string>>();
 
+  constructor(private readonly onProbeChange?: () => void) {}
```

Comment:
rather than exposing an onChange callback, which feels a bit of a hack can we just expose a generation counter, (incremented on every change), so you can use that in recording_manager.ts without injecting callbacks, just by checking the gen